spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.props;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
```

Comment:
Is this class still used, other than by the Constants class?
If you decide that you no longer need the Constants class, then do you still need this one?